modules/grandpa/src/weights.rs

```diff
@@ -69,29 +60,29 @@ pub struct RialtoWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for RialtoWeight<T> {
 	fn submit_finality_proof(v: u32, p: u32) -> Weight {
 		(0 as Weight)
-			.saturating_add((160_060_000 as Weight).saturating_mul(v as Weight))
-			.saturating_add((640_223_000 as Weight).saturating_mul(p as Weight))
-			.saturating_add(T::DbWeight::get().reads(5 as Weight))
-			.saturating_add(T::DbWeight::get().writes(3 as Weight))
```

Comment:
@HCastano so what I meant [here](https://github.com/paritytech/parity-bridges-common/pull/890#issuecomment-818829027) is this - current benchmarks are not doing pruning. These two additional writes are writes to `ImportedHashes` and `ImportedHashesPointer`. But there's no additional write for header removal (`<ImportedHeaders<T, I>>::remove(hash);`). I suggest to change benchmarks to include that.